README.md

```diff
@@ -54,9 +54,25 @@ $ npm run watch
 ### Building docs from source
 
 ```bash
+$ cd assets
+$ npm install
+$ cd ..
 $ MIX_ENV=docs mix docs
 ```
 
+## Building phoenix from source
+```bash
+$ mix deps.get
+$ mix compile
+```
+
+## Building phoenix installer from source
+```bash
+$ mix deps.get
+$ mix compile
+$ mix archive.build
+```
+
```

Comment:
Instead of these two sections, can you merge these instructions with the `Generating a Phoenix project from unreleased versions` heading? Thanks!

Reply:
Done, let me know what you think